package.json

```
@@ -14,16 +14,11 @@
   "scripts": {
     "start": "yarn stripes serve",
     "build": "yarn stripes build --output ./output",
-    "test:unit": "jest --ci --coverage",
-    "test:e2e": "yarn stripes test karma",
-    "test": "yarn run test:unit && yarn run test:e2e",
+    "test": "jest --ci --coverage",
```

can we keep "test:unit" and make this as 
```suggestion
    "test": "yarn run test:unit",
```
?

---

yep, will update